### ISSUE TITLE
DirectOptionMenus seem to be broken

### ISSUE THREAD

**Issue by ArsThaumaturgis** [panda3d/panda3d#674]
It looks like there's a bug in the distributable version of DirectOptionMenu: at time of writing, distributable builds of apps using a DirectOptionMenu crash with the following message:
```KeyError: 'Unknown option "popupMarker_pos" for DirectOptionMenu'```

I've observed this under Ubuntu 18.04.2.

Below is a short test-program (with setup.py and requirements.txt files) that, on my machine, displays the issue. To test, build the app via the build-system, and attempt to run it. It should crash, and print to the command-line the error shown above.

The app.
```
from direct.showbase import ShowBase as showBase

from direct.gui.DirectGui import DirectOptionMenu

class Game(showBase.ShowBase):

    def __init__(self):
        showBase.ShowBase.__init__(self)

        self.accept("escape", self.userExit)

        self.menu = DirectOptionMenu(
            scale = 0.1,
            pos = (0, 0, 0),
            items = ["Kittens", "Cats", "Mew"],
        )

app = Game()
app.run()
```

The "setup.py" file. The "gui_apps" entry assumes that the above source-file is named "bugHunt.py"; change this as appropriate.
```
from setuptools import setup

setup(
    name = "OptionMenuTest",
	version = "1.0.0",
    options = {
        "build_apps" : {
            "include_patterns" : [
            ],
            "include_modules" : {
            },
            "gui_apps": {
                "OptionMenuTest" : "bugHunt.py"
            },
            "plugins": [
                "pandagl"
            ],
            "platforms": [
                "manylinux1_x86_64",
                'win32',
                'win_amd64'
            ],
        }
    }
)
```

The "requirements.txt" file
```
panda3d
```

**Comment by ArsThaumaturgis** [comment]
It looks like this should be fixed in https://github.com/panda3d/panda3d/pull/662. If so, then I imagine that it's just a matter of that fix getting into the distributables!

**Comment by ArsThaumaturgis** [comment]
Update:

I just updated my Panda SDK to 1.10.3, and it looks like this issue is present there, too. :/

**Comment by Moguri** [comment]
Unfortunately, it looks like that fix (387b43adde75184ddb9801ee05188e0cc09da994) is not in 1.10.3. If you just want to do some testing of the distribution and do not mind using non-optimized builds, you can use dev builds of Panda.

To do this, update your requirements.txt file to:
```
--dev --extra-index-url https://archive.panda3d.org/branches/master
panda3d
```

And set `use_optimized_wheels` to `False` under your `build_apps` options in your `setup.py`. More information on optimized wheels can be found [here](http://www.panda3d.org/manual/?title=Building_Binaries#Optimized_Builds).

**Comment by rdb** [comment]
Closing as duplicate, fixed in 387b43adde75184ddb9801ee05188e0cc09da994 .

There's an easy workaround: just pass in a `popupMarker_pos=` argument in the constructor.

**Comment by ArsThaumaturgis** [comment]
That workaround doesn't seem to work for me: adding the keyword-parameter "popupMarker_pos = (0, 0, 0)" to the construction of the DirectOptionMenu in the above program produces the same crash on my machine, it seems.

However, fair enough on this being a duplicate!
